src/components/pages/thankYouPage/ThankYouPage.tsx

```diff
@@ -1,41 +0,0 @@
-import React, { Component } from 'react';
-import { FormattedMessage } from 'react-intl';
-import { Link } from 'react-router-dom';
-import { Button } from 'reactstrap';
-import Layout from '../../layout/Layout';
-import ThankYouLegend from '../../legends/thankYouLegend/ThankYouLegend';
-import './thankYouPage.scss';
-
-interface Props {
-  resetValues: Function;
-  resetBerths: Function;
-  resetWinterAreas: Function;
-  resetApplication: Function;
-}
-
-class ThankYouPage extends Component<Props> {
-  constructor(props: Props) {
-    super(props);
-
-    window.scrollTo(0, 0);
```

Comment:
> window.scrollTo(0, 0);

Should the `NoticeTemplate` also perform this "scroll to top" behavior?

Reply:
It did, but thanks to the suggestion I made sure to also wrap it in `useEffect`